querybook/server/logic/lineage.py

```diff
@@ -0,0 +1,202 @@
+from sqlalchemy import func, and_
```

Comment:
can we put db logic in logic.lineage and have a separate file for DATA_LINEAGE_BACKEND? since this is in logic, it might be confusing to people why this file is not directly used, but imported in metastore

Reply:
Not sure I understand what you mean here. Does this still apply after I moved the code around per other comments?